[2022-08-23 11:32:10,325] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-09-28T17:00:00+00:00 [queued]>
[2022-08-23 11:32:10,342] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-09-28T17:00:00+00:00 [queued]>
[2022-08-23 11:32:10,342] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:32:10,343] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-23 11:32:10,343] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:32:10,365] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): print_world> on 2021-09-28 17:00:00+00:00
[2022-08-23 11:32:10,374] {standard_task_runner.py:52} INFO - Started process 2288 to run task
[2022-08-23 11:32:10,380] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'raizen_lab_01', 'print_world', 'scheduled__2021-09-28T17:00:00+00:00', '--job-id', '814', '--raw', '--subdir', 'DAGS_FOLDER/raizen_lab_01.py', '--cfg-path', '/tmp/tmpp7f2xu61', '--error-file', '/tmp/tmplp2286xt']
[2022-08-23 11:32:10,380] {standard_task_runner.py:80} INFO - Job 814: Subtask print_world
[2022-08-23 11:32:10,501] {task_command.py:371} INFO - Running <TaskInstance: raizen_lab_01.print_world scheduled__2021-09-28T17:00:00+00:00 [running]> on host 7f48b7aaf0fc
[2022-08-23 11:32:10,624] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=raizen_lab_01
AIRFLOW_CTX_TASK_ID=print_world
AIRFLOW_CTX_EXECUTION_DATE=2021-09-28T17:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-28T17:00:00+00:00
[2022-08-23 11:32:10,626] {logging_mixin.py:115} INFO - world
[2022-08-23 11:32:10,626] {python.py:173} INFO - Done. Returned value was: None
[2022-08-23 11:32:10,650] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=raizen_lab_01, task_id=print_world, execution_date=20210928T170000, start_date=20220823T113210, end_date=20220823T113210
[2022-08-23 11:32:10,711] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-23 11:32:10,758] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
